libnet/src/ioctl.rs

```diff
@@ -752,19 +779,19 @@ fn plumb_for_af(name: &str, mut ifflags: u64) -> Result<(), Error> {
 
     // pop off unwanted modules
     loop {
-        if unsafe { rioctl!(mux_fd, sys::I_POP, 0).is_err() } {
+        if unsafe { rioctl_ro!(mux_fd, sys::I_POP, 0).is_err() } {
```

Comment:
I can kinda sorta see why this works, but I wonder if you might want to pass `ptr::null()` here, and similarly in other places where the third-argument is nil? Optionally, perhaps a macro variant that omits the third argument entirely?

Reply:
Good call, I've made it so the `ioctl_ro!` macro can be called with only two arguments.

Reply:
As long as those specific ioctls don't actually require a third argument, that looks good to go. :-)